xcodeproj/internal/templates/runner.sh

```diff
@@ -88,18 +88,26 @@ if [[ -z "$bazel_path" ]]; then
   exit 1
 fi
 
+# Check that either `tools/bazel` or `Tools/bazel` exist
+bazel_wrapper_path=""
+if [[ -f "tools/bazel" ]]; then
+  bazel_wrapper_path="./tools/bazel"
+elif [[ -f "Tools/bazel" ]]; then
+  bazel_wrapper_path="./Tools/bazel"
+fi
+
 if [[ "${BAZEL_REAL:-}" == "$bazel_path" && \
       -n "${BAZELISK_SKIP_WRAPPER:-}" && \
-      -f "tools/bazel" ]]
+      -n "${bazel_wrapper_path:-}" ]]
```

Comment:
Since we always assign to this, we don't need the conditional expansion:
```suggestion
      -n "$bazel_wrapper_path" ]]
```

Reply:
Ah i thought an empty string is still undefined? Similar to `foo=()`

Reply:
nvm just tested, and its fine